### PR TITLE
Creating the release note for Sept docs changes

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Creating the release note for Sept changes in the product docs. Internal link [here](https://www.notion.so/dbtlabs/September-docs-updates-8d1c970390a94f9a9d3c6956481ff3fa?pvs=4)

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Comment by runleonarun** [comment]
@dbt-labs/product-docs

File: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- Added [Postgres PrivateLink set up page](/docs/cloud/secure/postgres-privatelink)
+- Beta docs for [dbt Explorer](/docs/collaborate/explore-projects)
+- Added a new Semantic Layer [GraphQL API doc](/docs/dbt-cloud-apis/sl-graphql) and updated the [integration docs](/docs/use-dbt-semantic-layer/avail-sl-integrations) to include Hex. Responded to dbt community feedback and clarified Metricflow use cases for dbt Core and dbt Cloud.
+- Added an [FAQ](/faqs/Git/git-migration) describing how to migrate from one git provider to another in dbt Cloud.
+- Clarified an example and added a [troubleshooting section](/docs/cloud/connect-data-platform/connect-snowflake#troubleshooting) to Snowflake connection docs to address common errors and provide solutions.
+
+
+## 🎯 Core projects
+
+- Deprecated dbt Core 1.0 and 1.1 from the docs.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md]:
```suggestion
- Deprecated dbt Core v1.0 and v1.1 from the docs.
```

File: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Added [AWS Glue, community plugin](/docs/core/connect-data-platform/glue-setup)
+- Revised the dbt Core quickstart, making it easier to follow. Divided this guide into steps that align with our [other guides](/quickstarts/manual-install?step=1)
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md]:
```suggestion
- Revised the dbt Core quickstart, making it easier to follow. Divided this guide into steps that align with the [other guides](/quickstarts/manual-install?step=1).
```

File: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## New 📚 Guides, ✏️ blog posts, and FAQs
+
+Added a [style guide template](/guides/best-practices/how-we-style/6-how-we-style-conclusion#style-guide-template) that you can copy & paste to make sure you adhere to best practices when styling dbt projects!
+
+## Upcoming changes
+
+Stay tuned for our flurry of releases in October and a filterable guides section that will make our guides easier to find!
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md]:
```suggestion
Stay tuned for a flurry of releases in October and a filterable guides section that will make guides easier to find!
```

File: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+tags: [Sept-2023, product-docs]
+date: 2023-10-10
+sidebar_position: 09
+---
+
+Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 15 new community contributors to docs.getdbt.com. We merged [107 PRs](https://github.com/dbt-labs/docs.getdbt.com/pulls?q=is%3Apr+merged%3A2023-09-01..2023-09-31) in September.
+
+Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/):
+
+* Migrated docs.getdbt.com from Netlify to Vercel
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md]: Should we expound upon this?
```suggestion
* Migrated docs.getdbt.com from Netlify to Vercel.
```

File: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## ☁ Cloud projects
+- Continuous integration jobs are now generally available and no longer in beta!
+- Added [Postgres PrivateLink set up page](/docs/cloud/secure/postgres-privatelink)
+- Beta docs for [dbt Explorer](/docs/collaborate/explore-projects)
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md]:
```suggestion
- Published beta docs for [dbt Explorer](/docs/collaborate/explore-projects).
```

File: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- Added a new Semantic Layer [GraphQL API doc](/docs/dbt-cloud-apis/sl-graphql) and updated the [integration docs](/docs/use-dbt-semantic-layer/avail-sl-integrations) to include Hex. Responded to dbt community feedback and clarified Metricflow use cases for dbt Core and dbt Cloud.
+- Added an [FAQ](/faqs/Git/git-migration) describing how to migrate from one git provider to another in dbt Cloud.
+- Clarified an example and added a [troubleshooting section](/docs/cloud/connect-data-platform/connect-snowflake#troubleshooting) to Snowflake connection docs to address common errors and provide solutions.
+
+
+## 🎯 Core projects
+
+- Deprecated dbt Core 1.0 and 1.1 from the docs.
+- Added [AWS Glue, community plugin](/docs/core/connect-data-platform/glue-setup)
```
Comment (matthewshaver) [inline website/docs/docs/dbt-versions/release-notes/03-Oct-2023/product-docs-sept-rn.md]:
```suggestion
- Added configuration instructions for the [AWS Glue](/docs/core/connect-data-platform/glue-setup) community plugin.
```

**Review by mirnawong1 — APPROVED** [review]
lgtm!

**Pull request merged**